### DISCUSSION TITLE
Terrariumpi

### DISCUSSION THREAD

**Discussion by vivkid** [theyosh/TerrariumPI#578] [General]
Is there someone out there that can help me as I am trying to build a 125 gallon bio active enclosure and want everything controlled by a raspberry pi but I want it to where if the humidity is too low it will turn my lights down to a purple blue light with lightning effect and thunder sounds with a mister  and possibly fans and cloud effect on my lights with day and night dimmers with moon light and if temp is too low it will turn on a ceramic heat bulb or if too hot or too much humidity to kick fans on I have no experience with a pi is there a tutorial video on how to and all the hardware I would need and how to put it together and what I would need to run something that Big as in lights mister and fans that would also work for the pi by the way it would be for dart frogs any help is greatly appreciated and hey if someone already has one built where I can just buy my own  lights mister fan and heat bulb plug them in and set it up money is not a problem

**Comment by theyosh** [comment]
Hi,

out of the box, your light setup is not possible. But you can make it work with some small extra scripts.

So, that means you need some scripting skills. And you state that you are not familiar with a PI, so I am afraid that it is a bit hard to make it all work alone. So, you need to find someone close to you how is willing to help. Needs some level of programming skill.

We can do some suggestions on how to fix stuff, but I cannot make it from scratch for you. That is just to much work and time.
